libraries/validate/test/validate_test.js

```diff
@@ -80,19 +80,15 @@ suite('Valid Schema Tests', () => {
   });
 
   test('using constants.yml', async () => {
-    try {
-      let v = await validator({
-        folder: 'test/schemas',
-        constants: 'test/schemas/constants.yml',
-        serviceName: 'whatever',
-      });
-      let error = v(
-        {value: 43},
-        libUrls.schema(rootUrl, 'whatever', '/v1/yml-test-schema#'));
-      return assert.equal(error, null);
-    } catch (err) {
-      return err;
```

Comment:
wat

Reply:
This blames to me but I had the same reaction when I saw it.